care/emr/resources/facility/spec.py

```
@@ -28,12 +35,27 @@ class FacilityBaseSpec(FacilityBareMinimumSpec):
+
+DISCOUNT_CODE_COUNT_LIMIT = 100
+DISCOUNT_MONETARY_COMPONENT_COUNT_LIMIT = 100
 
 
 class FacilityCreateSpec(FacilityBaseSpec):
     geo_organization: UUID4
     features: list[int]
+    invoice_number_expression: str | None = None
+
+    @field_validator("invoice_number_expression")
+    def validate_invoice_number_expression(cls, v):
```

Ruff (0.11.9)

First argument of a method should be named `self`

Rename `cls` to `self`

(N805)